### ISSUE TITLE
[1.19.2] - (1.19-1.0.107) fps drop when mining a block with a pickaxe

### ISSUE THREAD

**Issue by DankestO** [ldtteam/Domum-Ornamentum#164]
This only happens with the pickaxes, with the other tools nothing happens. This gets worse if you go fast breaking blocks. The same thing happens with and without shaders. 
edit: I just realized the same thing happens with the other tools.

With the other tools it takes longer and I can pick up the block, but with the pickaxe it takes less, but the block disappears and I can't pick it up.

![test 1](https://github.com/ldtteam/Domum-Ornamentum/assets/126206944/deb49a96-5259-4a76-bdfa-fb68ed242b23)
![test 2](https://github.com/ldtteam/Domum-Ornamentum/assets/126206944/771c022f-de0f-4a65-9919-b2cee8f79cbc)

**Comment by Raycoms** [comment]
Could you run spark while doing this?

**Comment by someaddons** [comment]
and add the latest.log from during the issue pls^^

**Comment by Raycoms** [comment]
And please try the newest version too

**Comment by DankestO** [comment]
I don't quite know how to use spark, but I hope what I did helps.
I tried this in singleplayer, despite the fact that I discovered the failure on a dedicated server, even in this case the fps dropped even more.
This time it was with an axe, with a block made of wood in both slots.

btw, I also installed the latest version as you told me: 1.19-1.0.111

https://spark.lucko.me/S6Jx0g9cqR

[latest.log](https://github.com/ldtteam/Domum-Ornamentum/files/12252958/latest.log)
